.gitignore

```diff
@@ -13,8 +13,6 @@ gen/
 /example/basic/basic
 /example/grpc/client/client
 /example/grpc/server/server
```

Comment:
```suggestion
```

Since we don't have the `gRPC` example anymore, should we remove these lines too?

Reply:
Thanks for the review. Done and updated the commit message as well.